LQResultProducer.py

```diff
@@ -4069,6 +4090,13 @@ def ModSelection(selection,sysmethod,channel_log):
 		if sysmethod == 'PUdown':
 			selection = selection.replace('weight_central','weight_pu_down')
 
+		if sysmethod == 'BTAGSFup':
+			if year == '2016':
+				selection = selection.replace('*(((DeepJet_jet1>0.3093)+(DeepJet_jet2>0.3093))>0)',bTagSFmediumUp)
```

Comment:
This one I don’t get. It looks like you are replacing the Btag cut with the up/down SF variation, instead of the scale factor itself